Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverControllerFinder.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+#if UNITY_2017_2_OR_NEWER
+using UnityEngine.XR.WSA.Input;
+#else
+using UnityEngine.VR.WSA.Input;
+#endif
+using HoloToolkit.Unity.InputModule;
+
+namespace HoloToolkit.Unity
+{
+    public class SolverControllerFinder : MonoBehaviour
```

Comment:
Pretty sure there's already a class that should help find the controllers.

Reply:
There is, but its folded into AttachToObject. If we abstracted the attaching functionality from the controller event logic I could axe SolverControllerFinder.